### PR TITLE
Fix CWE-563 & discoverymgr testing

### LINKED ISSUE #171: Mitigating security vulnerability analysis from LGTM
_Originally related to PR #170 proposed by @tdrozdovsky_

### PR DESCRIPTION
**Pull request by tdrozdovsky**
Signed-off-by: Taras Drozdovskyi <t.drozdovsky@samsung.com>

# Description

Fix 2 alerts from LGTM and fix testing the discoverymgr

Fixes #171 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

The result can be seen after accepting the code in the master branch and restarting the LGTM analyzer

also you will be able to successfully run local test discoverymgr

```
$ ./build.sh test discoverymgr
```

**Test Configuration**:
* Firmware version: OS type and version, e.g., Ubuntu 16.04
* Hardware: x86-64
* Toolchain: Docker and Go recommended versions
* Edge Orchestration Release: Coconut

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/controller/discoverymgr/discovery_test.go [hunk 1]
```diff
@@ -18,7 +18,6 @@
 package discoverymgr
 
 import (
-	"log"
```
Comment (MoonkiHong) [inline src/controller/discoverymgr/discovery_test.go:21]: Any reason that we need to remove `log` from `discoverymgr`, in order to fix its testing?

**Review by MoonkiHong — COMMENTED** [review]
@tdrozdovsky Please take a look at my comment.

**Review by MoonkiHong — APPROVED** [review]
LGTM.

**Review by t25kim — APPROVED** [review]
LGTM! Thank you!

File: src/controller/discoverymgr/discovery.go [hunk 2]
```diff
@@ -19,8 +19,8 @@ package discoverymgr
 
 import (
 	"bufio"
-	"io/ioutil"
 	"common/logmgr"
+	"io/ioutil"
```
Comment (suresh-lc) [inline src/controller/discoverymgr/discovery.go:23]: Any reason for changing the order

**Review by suresh-lc — APPROVED** [review]
LGTM

**Pull request merged**